app/models/employment.rb

```diff
@@ -1,12 +1,22 @@
-class Employment < ApplicationRecord
-  belongs_to :assessment
+class Employment
+  include ActiveModel::Model
+  include ActiveModel::Attributes
 
-  has_many :employment_payments, dependent: :destroy
+  attribute :name, :string
+  attribute :client_id, :string
+  attribute :type, :string, default: "ApplicantEmployment"
```

Comment:
don't think we need this type field - it was used for Single Table Inheritance in the DB